### ISSUE TITLE
Support vgo 

### ISSUE THREAD

**Issue by ghost** [dave/jsgo#5]
Very very cool.

It's going to be interesting how the new golang packaging versioning plays out from RSC. It would allow a one to one mapping of semvar versioning.
Just an idea.

**Comment by dave** [comment]
Yep. Supporting `vgo` (or whatever it turns into) is definitely on my roadmap, however it will be a big undertaking. I created an issue in the go tracker asking for a vgo feature allowing code to do a "go get". Right not the only interface is from the command line, so it would involve forking internal packages. Hopefully with production vgo, it won't be as painful as it was this time (see https://github.com/dave/jsgo/tree/master/getter)! 

Here's the issue: https://github.com/golang/go/issues/23951
